packages/web-api-scan-runner/src/scan-runner-telemetry-manager.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { inject, injectable } from 'inversify';
+import { isNil } from 'lodash';
+import { GlobalLogger, ScanTaskCompletedMeasurements } from 'logger';
+
+@injectable()
+export class ScanRunnerTelemetryManager {
+    public scanSubmitted: number;
```

Comment:
These probably shouldn't be public because their only public use is unit testing. The problem is essentially one of reading the code: When I first read this class I automatically assumed that because these properties are public, they are required by other production code.
One way you could fix this is to make the values protected; and in the test file create a class which inherits from ScanRunnerTelemetryManager and exposes access to them. (That may not be the only way option.)